content/chainguard/chainguard-images/troubleshooting/debugging-distroless-images.md

```diff
@@ -22,13 +22,13 @@ Because distroless images are minimal and don't include a package manager or a s
 
 In this article, we'll discuss a few different strategies to debug distroless images.
 
-## 1. Using Dev / Debug Image Variants
+## 1. Using Development Container Image Variants
 
 Before moving a workload to a distroless runtime image, it is important to make sure that it runs without issues in a similar but less restrictive environment, which allows for easier debugging. It is also possible to make a temporary base image change from a distroless image to a fully featured image that offers more debugging capabilities.
 
-The `-dev` variants of [Chainguard Containers](/chainguard/chainguard-images/) are designed to replicate the same packages of their distroless version, but with additional software that helps in developing, building, and debugging applications in different language ecosystems.
+The development variants of [Chainguard Images](/chainguard/chainguard-images/) are designed to replicate the same packages of their distroless version, but with additional software that helps in developing, building, and debugging applications in different language ecosystems. These are sometimes referred to as `-dev` variants since they are tagged with `:latest-dev`.
```

Comment:
```suggestion
The development variants of [Chainguard Containers](/chainguard/chainguard-images/) are designed to replicate the same packages of their distroless version, but with additional software that helps in developing, building, and debugging applications in different language ecosystems. These are sometimes referred to as `-dev` variants since they are tagged with `:latest-dev`.
```

Reply:
Sorry, I think some of your changes got lost when I tried syncing my local branch with the latest changes